OOPP/app/build.gradle

```diff
@@ -20,6 +20,11 @@ android {
     }
 }
 
+android {
+    testOptions {
+        unitTests.returnDefaultValues = true
```

Comment:
oj kommenterade fel - detta var det som enable:ade mina tests. det stod i den linken jag fick att det man kunde lägga till detta i build.gradle för att förhindra "not mocked"-error. 
Scrolla ner på denna: https://developer.android.com/training/testing/unit-testing/local-unit-tests.html